src/media/js/main.js

```diff
@@ -121,8 +121,26 @@ require(
 
         z.body.toggleClass('logged-in', user.logged_in());
         z.page.trigger('reloaded_chrome');
+    }).on('install-success reload_chrome', function() {
```

Comment:
It needs to run on each page so I think so. It could go in a different file but I feel like main is where someone would look for this (it's where all the other banner code is initialized).